app/scripts/controllers/preferences.js

```diff
@@ -338,7 +338,7 @@ class PreferencesController {
     if (_url !== 'http://localhost:8545') {
       rpcList.push(_url)
     }
-    if (rpcList.length > 2) {
```

Comment:
Could you update the documentation for this method to reflect the change in the max length of the rpcList?